com.microsoft.mrtk.tools/SubsystemWizard/Templates/SubsystemInterfaceTemplate.txt

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// TODO: [Optional] Add additional copyright statement(s).
+
+namespace %NAMESPACE%
+{
+    /// <summary>
+    /// Interface defining the functionality of the #SUBSYSTEMBASECLASSNAME#.
```

Comment:
It seems like `#TEMPLATE#` and `%TEMPLATE%` are mixed and matched. Are there supposed to be two different template symbols/formats?

Reply:
there should be only one.... my error

Reply:
fixed